### ISSUE TITLE
Issue with Products Archive

### ISSUE THREAD

**Issue by vbasauth** [elementor/hello-theme#155]
I am using the "Hello Theme" because the default Wordpress theme was causing major issues with formatting for my products.  Your theme seemed to fix my main issues.  But when I try to edit my products archive, I make changes in the Elementor editor -- visibly able to do so -- and they don't seem to apply to the live site (once published).  I'm attaching two photos to show what I'm talking about.  I'm not sure why things look fine in the editor, but they don't look fine in the live site.  I want things to be centered, I want the price to look a certain way, and I want my "Add to Cart" button to be brown.  I'm posting this here because I'm assuming that this might be an error (as I'm using an Elementor theme with the Elementor editor).  

![Undesired](https://user-images.githubusercontent.com/84265928/118391114-4a6b6080-b5f8-11eb-979b-534519c2020c.png)
![Desired](https://user-images.githubusercontent.com/84265928/118391116-4b03f700-b5f8-11eb-802a-5e3f526b5ece.png)

**Comment by gilesholder** [comment]
Have you tried globally changing the button styling in global options?

**Comment by vbasauth** [comment]
Yes.  It's a bigger problem than just the button -- notice the centering as well.  I found that I was able to make my changes visible when I set my products page to a "shop page" in WooCommerce settings, but the problem with doing this is that I was no longer able to edit the page with Elementor.  Supposedly Elementor cannot edit a "shop page" (I've tried doing so with several pages that I turned into a "shop page" and all of them were unable to be edited once the conversion was done).  So I had to go a different route.  I essentially created a product loop and inserted my product archive into my products page by using the template widget.  This essentially solved my problem above and works fine.

As for my experience with Elementor, I can't say I'm all that happy with it.  I've spent weeks trying to tackle numerous problems.  Certain settings seem out of reach -- even when I add other plug-ins.  I'm pretty much "settling" with Elementor because I don't want to pay Wix prices (and the like).  I just feel that Elementor is sorely lacking in some areas.  Having to buy other plugins to help cover its deficiencies just helps inch me closer to paying as much as I would for a Wix site (or Dreamweaver or whatever).  Despite downloading a CSS plugin, I'm having problems with my cart and checkout pages in the Edge browser (table formatting).  I miss my days with Frontpage.

But then this is also a fundamental problem with Wordpress - having to use an underlying theme and then having to brutally warp it and shape it into what I want it to be.  This is a completely inelegant way to design a website.  I'm sure there are people who seriously lack talent and will be happy to settle for a theme as is, but none of the themes I saw looked remotely the way I wanted, and most were clogged up with all kinds of splashy garbage.

One other problem I had with the "Hello" theme is that certain colors show up when you add items to the cart or when the fields are filled in on the checkout page.  The lime green color clashes with my color scheme.  I think I've also seen blue show up on occasion.  I guess red is okay (and necessary).  But I can't seem to find a way to change these colors.  Why aren't these colors listed as settings in the settings panel?  Also, I like the "Add to Cart" button with the shopping cart icon on it, but I can only use it on my archive page (defined in the product loop) but I can't use the same button on actual individual product pages -- IF I want to allow users to change the amount of product to order.  I've tried to add the "Font Awesome" cart to the regular "add to cart" button, but it won't appear without having to go through some convoluted steps.  I also tried adding the cart via the Wordpress set of icons, but again, the process is too convoluted (I don't care to mess with PHP files).  Hassle after hassle after hassle....

**Comment by diegoALCE95** [comment]
Just tested with an E-commerce that I'm doing and it works fine.
